### ISSUE TITLE
Feature req

### ISSUE THREAD

**Issue by shunrexd** [ToxSylph/SeaOfChoros#7]
Do you have any chance to add riddle esp?

**Comment by ToxSylph** [comment]
Sorry, are you talking about the gold hoarders maps that give you clues to dig for treasure? If that is the case, you can go to 
Object Esp > Islands > Show Rare Spots. This will mark all the clue spots with an "X", then you can use the Filter Keyword textbox to filter by key words (single words described by the clue, in English) to get the clue spot that you want. I hope this feature  is usefull.

**Comment by shunrexd** [comment]
> Sorry, are you talking about the gold hoarders maps that give you clues to dig for treasure? If that is the case, you can go to Object Esp > Islands > Show Rare Spots. This will mark all the clue spots with an "X", then you can use the Filter Keyword textbox to filter by key words (single words described by the clue, in English) to get the clue spot that you want. I hope this feature is usefull.

Yes gold hoarders maps but its not X Spot i mean riddle like this;
![image](https://user-images.githubusercontent.com/72516739/206702576-6a54b23e-4e1c-40a8-86de-7fef8317b0df.png)

and there is an explanation https://www.unknowncheats.me/forum/sea-of-thieves/487535-reading-riddles.html
